source/includes/fact-oplog-background.rst

```diff
@@ -1,7 +1,15 @@
-The :term:`oplog` in the source cluster must be large enough to track
-events that happen during the time it takes to complete the initial
-sync to the destination cluster.
+
+``mongosync`` applies operations in the ``oplog`` on the source cluster
+to the data on the destination cluster.  When older operations 
```

Comment:
(suggestion)
"older" here is a little vague and might be confusing. I think you could just omit, on the grounds that "operations that ``mongosync`` has not applied" is what you really mean. Please holler at me if I'm missing the point here.

Reply:
Fixed